src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -234,6 +235,13 @@ class DiskImageEditor extends Component {
     }
   }
 
+  changeBootable (event) {
+    const target = event.target
+    const value = target.type === 'checkbox' ? target.checked : target.value
```

Comment:
Is this condition really needed? Because `changeBootable` is used only for `Checkbox`, so value always will be in `target.checked`